config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/WelcometothePlas-bNiieqPgSFqwFpC2DZkj-g==.json

```diff
@@ -0,0 +1,55 @@
+{
+  "questIDHigh:4": 7843197398684813402,
+  "preRequisites:9": {
+    "0:10": {
+      "questIDHigh:4": 0,
+      "questIDLow:4": 1212
+    }
+  },
+  "questIDLow:4": -5758255961960012806,
+  "properties:10": {
+    "betterquesting:10": {
+      "snd_complete:8": "random.levelup",
+      "taskLogic:8": "AND",
+      "visibility:8": "NORMAL",
+      "isMain:1": 0,
+      "simultaneous:1": 0,
+      "icon:10": {
+        "id:8": "gregtech:gt.metaitem.01",
+        "Count:3": 1,
+        "Damage:2": 30584,
+        "OreDict:8": ""
+      },
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "Welcome to the Plasma Chain",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "Hello there! If you\u0027ve unlocked fusion, you can start this questline, but keep in mind that it\u0027s divided into three segments (the three fusion chains): one for each tier of Fusion Reactor, from Mk1 to Mk3.\n\nThere is a spreadsheet for all these chains, although it has some reactor count numbers that aren\u0027t right. To see it, check the PR that set these numbers: §[§1[url]https://github.com/GTNewHorizons/GTplusplus/pull/669[/url]§r\n\nThese quests follow the process of creating some truly powerful plasmas that are useful for power generation at the beginning of endgame, when you\u0027ve scaled up your setup to a Compact Fusion Reactor and XL Plasma Turbines. As you might have seen before, those huge turbines can go very fast with good materials, like Trinium or Infinity, but they will apply EU/t penalties according to the fuel value of the plasma used. This means that Helium and Tin Plasma are both much weaker there than they were in Large Plasma Turbines.\n\nThe end product of this whole process is Celestial Tungsten Plasma, which has no penalties even at Spacetime and can somewhat match what Helium Plasma would have produced, per compact reactor. The last step is the slowest, but, if there are no other bottlenecks, you can expect around §a§l1,093.5 billion EU/t§r. That is what a single compact, doing the last step, can give you with Infinity XLs. However, if you only use one Mk3 reactor per step until the last one, your compact will only run at 1/9 speed and produce §e121.5 million EU/t§r.\n\nHowever, getting there won\u0027t be easy. This whole process uses 15 fusion recipes in total, and a lot more material automation than any of the simple fusion setups. This questline will help explain what is simple, not simple, or possibly a bottleneck. If you complete this, you will be sustained in powergen until the Dyson Swarm, or whatever other option appears later on, and it will continue being useful later on!"
```

Comment:
- please use the standard format for the link.
- I dont think we can expect the player to know what a PR is. A wiki link would be best but if you are sticking with github then just say something like 'To see it, check on github where the numbers where set:'
- the 2 colored numbers are not great. the yellow is barely readable on the default layout and one is bold while the other isnt

Reply:
I underlined the link, changed the text to not mention PR, but instead a Github link, and I changed the colored numbers to be green and red, and also both bold.